src/Control/Lens/Plated.hs

```diff
@@ -444,7 +444,7 @@ cosmosOf d f s = f s *> d (cosmosOf d f) s
 -- @
 -- 'cosmosOn' :: 'Plated' a => 'Fold' s a -> 'Fold' s a
 -- @
-cosmosOn :: (Applicative f, Plated a) => LensLike' f s a -> LensLike' f s a
+cosmosOn :: Plated a => Fold s a -> Fold s a
```

Comment:
This is *worse*. It turns rank1types into rank2types. This has worse type-inference properties.